code/modules/shuttle/shuttle_creation/shuttle_creator.dm

```diff
@@ -290,6 +291,12 @@ GLOBAL_LIST_EMPTY(custom_shuttle_machines)		//Machines that require updating (He
 	for(var/door in firedoors)
 		var/obj/machinery/door/firedoor/FD = door
 		FD.CalculateAffectingAreas()
+
+	var/static/list/choice = list("NORTH" = NORTH, "SOUTH" = SOUTH, "EAST" = EAST, "WEST" = WEST)
+	var/Pdir = input(user, "Shuttle Fly Direction:", "Blueprint Editing", NORTH) as null|anything in list("NORTH", "SOUTH", "EAST", "WEST")
```

Comment:
Cant you just put NORTH WEST SOUTH EAST straight into the input list rather than doing this stuff with lists?

Reply:
And user see 1 2 4 8.

I first think use associated list instead of list("NORTH", "SOUTH", "EAST", "WEST")
but it don't work.

Reply:
Maby make it able change affter creation? I case that someone rebuild ship and want another heading transit dir?

Reply:
I see, change the default to "NORTH" instead of NORTH

Reply:
```suggestion
	var/Pdir = input(user, "Shuttle Fly Direction:", "Blueprint Editing", "NORTH") as null|anything in list("NORTH", "SOUTH", "EAST", "WEST")
```

Reply:
What about make it able change transit fly direction on will?
It just change visually fly direction.